pkg/types/types.go

```diff
@@ -6,6 +6,9 @@ import (
 	"github.com/pkg/errors"
 )
 
+// Address is an account address encoded using bech32 format
+type Address string
```

Comment:
Why do we use address as a string instead of sdk.Address, and convert in to the address anyway in the func call?